geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreTest.scala

```diff
@@ -72,6 +74,97 @@ class AccumuloDataStoreTest extends Specification with TestWithMultipleSfts {
       ds.getSchema(defaultSft.getTypeName) mustEqual defaultSft
     }
 
+    "create a schema with keywords" in {
+
+      val sftWithKeywords: SimpleFeatureType = createNewSchema("name:String", dtgField = None)
```

Comment:
this test could be simplified a lot if you just put the keywords at the end of the spec string. e.g.
`createNewSchema(s"name:String;$KEYWORDS_KEY='keywordA,keywordB,keywordC'")
Then you don't have to create a second schema